src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -84,7 +87,7 @@ export class EditDataGridPanel extends GridParentComponent {
 	public onCellEditEnd: (event: Slick.OnCellChangeEventArgs<any>) => void;
 	public onIsCellEditValid: (row: number, column: number, newValue: any) => boolean;
 	public onIsColumnEditable: (column: number) => boolean;
-	public overrideCellFn: (rowNumber, columnId, value?, data?) => string;
```

Comment:
why removing the rowNumber parameter?

Reply:
As I mentioned, it's only being called by loadItem, which does not have the rowNumber either, it was also not being used at all in the funciton